.taskcluster.yml

```diff
@@ -40,6 +40,8 @@ tasks:
           && git checkout {{event.head.sha}}
           && yes | sdkmanager --licenses
           && ./gradlew -PisPullRequest clean assembleSystem assembleAndroidTest lint checkstyle ktlint pmd detekt test
+          && ./gradlew -Pcoverage jacocoSystemDebugTestReport
+          && ./tools/taskcluster/upload-coverage-report.sh
```

Comment:
One downside to this invocation is that if TC fails to upload to codecov (e.g. codecov's servers are down), the whole build is going to fail. Preferably, this would be an optional step. However, I always forget how to build in optional steps so this seems fine. If it's a problem, we can change it later.